components/decimal/src/lib.rs

```diff
@@ -170,3 +166,35 @@ impl FixedDecimalFormatter {
         self.format(value).write_to_string().into_owned()
     }
 }
+
+icu_preferences::preferences!(
+    /// TODO
+    FixedDecimalFormatterPreferences,
+    ResolvedFixedDecimalFormatterPreferences,
+    {
+        /// TODO
+        numbering_system: NumberingSystem
+    }
+);
+
+impl FixedDecimalFormatterPreferences<'_> {
+    /// TODO
+    pub fn resolve(self) -> ResolvedFixedDecimalFormatterPreferences {
+        #[allow(clippy::unwrap_used)] // temporary
+        ResolvedFixedDecimalFormatterPreferences {
+            data_locale: DataLocale::from(self.lid),
+            numbering_system: self.numbering_system.unwrap(),
```

Comment:
Issue: it should just be stored as an Option, I think? We don't actually have a way to retrieve the resolved numbering system from data because that wasn't a design constraint until now. Maybe we should add it to DecimalSymbolsV2, #4437